### DISCUSSION TITLE
Extend `npm li` to filter by scope.

### DISCUSSION THREAD

**Discussion by Neutrino-Sunset** [npm/rfcs#228] [RRFC]
I'd like to be able to do `npm li @angular/ --depth=0` in order to verify that large libraries used in my application are using compatible versions.

This would then output only the version of the @angular packages.

**Comment by ruyadorno** [comment]
We discussed this item in the [OpenRFC meeting](https://github.com/npm/rfcs/blob/latest/meetings/2020-10-28.md#discussion-228-npm-ls-by-scope---neutrino-sunset) and the consensus seems to be that we want to support this usecase and it's def useful 😊 

Might be worth opening a proper RFC as next step just to confirm what syntax to use.
